data/src/main/java/io/gnosis/kouban/data/repositories/SafeRepository.kt

```diff
@@ -74,6 +74,39 @@ class SafeRepository(
             }
         }
 
+    suspend fun checkSafe(address: Solidity.Address): Pair<Solidity.Address?, Boolean> {
```

Comment:
we don't need this boolean. It is used to indicate whether safe is paired with browser extension. This is irrelevant for the prototype. checkSafe could just return boolean indicating it's a valid safe

Reply:
Are you sure about that? I think that is related to the threshold rather. If it is a multisig safe, then it still makes sense to check that we have at least one? I took this code from the safe, so is not like I am adding anything new.

Reply:
Yeah we do not need this here ;)

Reply:
Actually you are using this boolean in a wrong way here, because even a Safe with a threshold of 1 is a valid Safe